packages/modul-components/src/components/copy-to-clipboard/copy-to-clipboard.ts

```diff
@@ -79,6 +79,10 @@ export class MCopyToClipboard extends ModulVue {
         return { ...this.$props, ...new DefaultCopyToClipboardPropsValue(this.selectedText, this.value) };
     }
 
+    get showButton(): boolean {
+        return !(!!this.as<InputState>().disabled || !!this.as<InputState>().waiting);
```

Comment:
@chuckmah ça fait bizzare d'avoir à faire une passe passe comme ça (`!!this.as<InputState>().state`) lorqu'on utilise l'`InputState`. Si l'état n'est pas assigné, il retourne `undefined` par défaut. On devrait plutôt retourner l'état null par défaut en fonction du type (dans ce cas-ci `false`)